lib/cylc/task_events_mgr.py

```diff
@@ -1024,13 +1027,16 @@ def _reset_job_timers(self, itask):
                 default=[900]))  # Default 15 minute intervals
         try:
             itask.timeout = timeref + float(timeout)
-            timeout_str = intvl_as_str(timeout)
+            time_limit_str = intvl_as_str(time_limit)
         except (TypeError, ValueError):
             itask.timeout = None
-            timeout_str = None
+            time_limit_str = None
+
         itask.poll_timer = TaskActionTimer(ctx=ctx, delays=delays)
+
         # Log timeout and polling schedule
-        message = 'health check settings: %s=%s' % (timeout_key, timeout_str)
+        message = 'health check settings: %s=%s' % (
+            timeout_key, time_limit_str)
```

Comment:
I don't get the reason for this change

Reply:
@wxtim You marked this as resolved but didn't elucidate?

Reply:
I changed it back - this diff is marked outdated.

Unfortunately the actual reason for this change is that changing `timeout_str` to `time_limit_str` (personal preference, and timeout being used elsewhere for other purposes) has made this line too long.

Reply:
I'm not talking about the line wrapping - I'm talking about `timeout_str -> time_limit_str`

Reply:
I agree - I was trying to change the names to make it more readable, but I think I made a mistake with this.